edit_course/course_forms.py

```diff
@@ -91,8 +95,8 @@ class Meta:
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        self.fields["assistants"].widget.attrs["class"] = "search-select"
-        self.fields["assistants"].help_text = ""
+        self.fields['assistants'].queryset = self.instance.assistants.values_list('user', flat=True)
+        self.fields['assistants'].widget.attrs["data-search-api-url"] = api_reverse("user-list")
```

Comment:
I think the attr data-search-api-url can be defined in the UsersSearchSelectField constructor since it is the same for all user search forms. The queryset is kept here in the form class because it requires the course instance object.

Reply:
Actually, the form should define the API URL like it is now because some forms use the all users list from the API and some only use the enrolled students of one course.